### PR TITLE
Split gce-stable1-beta-upgrade jobs into serial and parallel jobs.

### PR DESCRIPTION
**Pull request by cjwagner**
Based on #10134 (same change for master jobs)

/assign @justinsb 
/cc @AishSundar

### CODE REVIEW

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml [hunk 1]
```diff
@@ -45,6 +69,32 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-zone=us-central1-f
       - --provider=gce
+      - --test_args=--ginkgo.focus=\[Serial\]|\[Disruptive\] --ginkgo.skip=\[Flaky\]|\[Feature:.+\]
```
Comment (cjwagner) [inline config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: Should this job have had the `--test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8` flag that the other jobs do?

**Comment by AishSundar** [comment]
This is for fixing https://github.com/kubernetes/kubernetes/issues/70912 

Thanks @cjwagner. Feel free to remove the hold once @justinsb reviews.

/lgtm
/hold
/milestone v1.13

@jberkus @mariantalla @mortent as FYI

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml [hunk 2]
```diff
@@ -20,7 +20,31 @@ periodics:
...
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --check-version-skew=false
+      - --env=STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
+      - --extract=ci/k8s-beta
+      - --extract=ci/k8s-stable1
+      - --gcp-node-image=gci
+      - --gcp-zone=us-central1-f
+      - --provider=gce
```
Comment (justinsb) [inline config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: We need `- --ginkgo-parallel` here I think

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml [hunk 3]
```diff
@@ -45,6 +69,32 @@ periodics:
...
```
Comment (justinsb) [inline config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: And here

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml [hunk 4]
```diff
@@ -70,10 +120,35 @@ periodics:
...
```
Comment (justinsb) [inline config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: And here :-)

**Comment by justinsb** [comment]
I think we missed the ginkgo-parallel flag on the parallel jobs (?)

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml (hunk 1 above)
Comment (justinsb) [reply config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: Sounds plausible - maybe open an issue so we can merge this PR without having to figure it out?

**Comment by justinsb** [comment]
(Otherwise lgtm)

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml (hunk 2 above)
Comment (cjwagner) [reply config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: That might help :man_facepalming:

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml (hunk 1 above)
Comment (cjwagner) [reply config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: https://github.com/kubernetes/test-infra/issues/10153

**Comment by cjwagner** [comment]
Comments addressed, PTAL.

**Pull request merged**